docs/how_to/point_manipulator.md

```diff
@@ -8,28 +8,29 @@ Precisely positioning a point in 3D is a useful way to annotate specific positio
 
 ## Instructions
 
-1. Open the [points_manipulator.py](https://napari-threedee.github.io/generated/gallery/plugin/points_manipulator_plugin/) example.
+1. To run this example, first download the script from the [Examples Gallery page](https://napari-threedee.github.io/generated/gallery/plugin/points_manipulator_plugin/); the link is at the bottom. Ensure the location is somewhere memorable. Or, if you've cloned this repository, the script can be found at `napari-threedee/docs/examples/plugin/points_manipulator_plugin.py`. 
 
+2. Ensure you have the appropriate virtual environment active, change directories to the script location, and then you can execute the script using:
+	```bash
+	python points_manipulator_plugin.py
+	``` 
 
-    ![example opened](https://user-images.githubusercontent.com/1120672/207381513-8fd3f7e9-157f-4ef9-99c1-551bd329842b.png)
-   The example opens in 3D rendering mode. Click and drag to rotate the view.
-   
-2. Open the point manipulator plugin from the Plugins menu: Plugins -> napari-threedee -> point manipulator. Click the "activate" to start the plugin.
+3. The viewer will open in 3D rendering mode with a Points layer and 3 points and the widget already visible. The widget can be otherwise accessed from the Plugins menu: **Plugins -> napari-threedee -> point manipulator**. Click the "activate" to start the plugin.
```

Comment:
```suggestion
3. The viewer will open in 3D rendering mode with a Points layer and 3 points and the widget already visible. The widget can be otherwise accessed from the Plugins menu: **Plugins -> napari-threedee -> point manipulator**. Click the "activate" button to start the plugin.
```